app/models/tslr_claim.rb

```diff
@@ -91,24 +93,21 @@ class TslrClaim < ApplicationRecord
   scope :submitted, -> { where.not(submitted_at: nil) }
 
   def submit!
-    if ineligible?
-      errors.add(:base, full_ineligibility_reason)
-      return false
-    end
-
-    if can_be_submitted?
+    if valid?(:submit)
```

Comment:
Is there a reason this isn't using `submittable?`?

Reply:
No reason at all. Not using it was allowing multiple submissions which we had no coverage for. 

Great spot, thanks @erbridge!